pkg/plugin/plugin.go

```diff
@@ -280,10 +319,30 @@ func CmdAdd(args *skel.CmdArgs) error {
 	}
 
 	var vlanTagNum uint = 0
+	var vlanMode = "access"
```

Comment:
How about using constants/enums?

I also suggest you introduce `vlanMode` in a separate commit and default it there to `access` or `trunk` per the inputted keys.